velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -462,6 +462,18 @@ class PlanBuilder {
       const std::vector<std::string>& aggregationInputs,
       std::string groupIdName = "group_id");
 
+  /// Add an ExpandNode using specified projections. See comments for
+  /// ExpandNode class for description of this plan node.
+  ///
+  /// @param projections A list of projection expressions. Each value in one
```

Comment:
typos: Each expression is either a column name, null or non-null constant.

Please, include examples of null projections and multiple rows.